rebar.lock

```diff
@@ -118,8 +118,8 @@
  {<<"syslog">>,{pkg,<<"syslog">>,<<"1.1.0">>},0},
  {<<"unicode_util_compat">>,{pkg,<<"unicode_util_compat">>,<<"0.7.0">>},1},
  {<<"vernemq_dev">>,
-  {git,"https://github.com/gojekfarm/vernemq_dev.git",
-       {ref,"1ab4a944cc2dd61e93ada1251964781a1389ebba"}},
+  {git,"https://github.com/VivekPipaliya23/vernemq_dev.git",
```

Comment:
Change back to stable gojekfarm ref.

Reply:
Updated [chore: update vernemq_dev version to stable](https://github.com/gojek/vernemq/pull/50/commits/27a885c3bd1d203181815870f0dbdda7e7eb36fe)